### ISSUE TITLE
Deck link on player profile and battles returning na for card ids

### ISSUE THREAD

**Issue by jimmyk3008** [cr-api/cr-api#220]
-->
```
GET
http://api.cr-api.com/player/20LQJGR9Q
```

## Response Details
```
Code: 200

{#369 ▼
  +"tag": "20LQJGR9Q"
  +"name": "Jimmy"
  +"trophies": 4376
  +"rank": null
  +"arena": {#367 ▶}
  +"clan": {#354 ▶}
  +"stats": {#356 ▶}
  +"games": {#353 ▶}
  +"chestCycle": {#361 ▶}
  +"leagueStatistics": {#371 ▶}
  +"deckLink": "https://link.clashroyale.com/deck/en?deck=n/a;n/a;n/a;n/a;n/a;n/a;n/a;n/a"
  +"currentDeck": array:8 [▶]
  +"cards": array:80 [▶]
  +"achievements": array:12 [▶]
  +"battles": array:25 [▶]
}

```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there? Recently
 2. Expected behaviour: scid of cards

**Comment by Mikesteinberg** [comment]
I am also having the same issue. Probably a result of the Latest API Deployment unless it's on Supercells' end.

**Comment by selfish** [comment]
Fixed in `v8.4.34`, thanks for the report.
